src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -21,13 +21,20 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Import Project="$(CustomBeforeMicrosoftCommonCrossTargetingTargets)" Condition="'$(CustomBeforeMicrosoftCommonCrossTargetingTargets)' != '' and Exists('$(CustomBeforeMicrosoftCommonCrossTargetingTargets)')"/>
 
   <Target Name="GetTargetFrameworks"
+          DependsOnTargets="GetTargetFrameworksWithPlatformFromInnerBuilds"
           Returns="@(_ThisProjectBuildMetadata)">
+
+    <Error Condition="'$(IsCrossTargetingBuild)' != 'true'"
```

Comment:
I like the assertion, we should have them everywhere, but I am morbidly curios how many people it will break :D

Reply:
If people want to call GetTargetFrameworks, I think they can do it from Microsoft.Common.CurrentVersion.targets. That sounds like a more legitimate place to call it if IsCrossTargetingBuild is false.

Reply:
I added these checks because previously the body of GetTargetFrameworks was the same in the CrossTargeting and CurrentVersion targets.  I assume that the CrossTargeting targets are only imported for the outer build, and the CurrentVersion targets only in an inner or  non-multi-targeted build.  The checks are basically assertions to verify that is correct.